doctors.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+class Doctors:
+    def __init__(self, base_url, login_url, email, password):
+        self.base_url = base_url
+        self.login_url = login_url
+        self.email = email
+        self.password = password
+        self.token = self.get_token()
+
+    def get_token(self):
+        headers = {
+            "Content-Type": "application/json"
+        }
+        data = {
+            "username": self.email,
+            "password": self.password
+        }
+        response = requests.post(self.login_url, headers=headers, json=data)
+
+        if response.status_code == 200:
+            return response.json().get("token")
+        else:
+            raise Exception("Failed to authenticate")
```

Comment:
Well we're not sure that's really the issue. So I wouldn't wrap it.
And the requests library has `raise_for_status()` that you could use.
https://requests.readthedocs.io/en/latest/api/#requests.Response.raise_for_status